docs/project-generation-options.rst

```diff
@@ -119,6 +119,7 @@ ci_tool:
     1. None
     2. `Travis CI`_
     3. `Gitlab CI`_
+    4. `Github CI`_
```

Comment:
Should we label this as Github Actions instead of Github CI?

Reply:
Sure, but I think having `CI` should make it immediately obvious in case someone hasn't heard of them before. 
But `Github CI` sounds fine too.